From 0a4c4caa1f175fde92b90cb8d86ca4d45768a289 Mon Sep 17 00:00:00 2001
From: Sabri Ünal <yakushabb@gmail.com>
Date: Sun, 28 Jun 2026 21:54:15 +0300
Subject: [PATCH] Fix a build error

> error: format not a string literal, argument types not checked [-Werror=format-nonliteral]
> var alert_dialog = new Gtk.MessageDialog((Gtk.Window) row.get_toplevel(),

Copied from: https://gitlab.gnome.org/GNOME/gitg/-/blob/master/libgitg/gitg-repository-list-box.vala#L557

Signed-off-by: Sabri Ünal <yakushabb@gmail.com>
---
 libgitg/gitg-repository-list-box.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libgitg/gitg-repository-list-box.vala b/libgitg/gitg-repository-list-box.vala
index 01bacbc..df54503 100644
--- a/libgitg/gitg-repository-list-box.vala
+++ b/libgitg/gitg-repository-list-box.vala
@@ -563,7 +563,7 @@ namespace Gitg
 		{
 			var alert_dialog = new Gtk.MessageDialog((Gtk.Window) row.get_toplevel(),
 					Gtk.DialogFlags.MODAL, Gtk.MessageType.ERROR, Gtk.ButtonsType.NONE,
-					_("Deleting repository source files “%s” will delete them from disk and cannot be undone. Are you sure?").printf(uri), null);
+					_("Deleting repository source files “%s” will delete them from disk and cannot be undone. Are you sure?"), uri);
 			alert_dialog.add_button(_("Cancel"), DeleteSources.CANCEL);
 			alert_dialog.add_button(_("Move to trash"), DeleteSources.TRASH);
 			alert_dialog.add_button(_("Delete permanently"), DeleteSources.DELETE);
--
libgit2 1.7.2
